src/hotspot/share/gc/parallel/mutableNUMASpace.cpp

```diff
@@ -53,7 +53,18 @@ MutableNUMASpace::MutableNUMASpace(size_t alignment) : MutableSpace(alignment),
     }
 #endif // LINUX
 
-  update_layout(true);
+  int lgrp_limit = (int)os::numa_get_groups_num();
```

Comment:
Pre-existing, but if you like you can make `lgrp_limit` `size_t`, one ugly cast can be removed. Unfortunately the cast two lines below can not be removed because of `int` indexing in growable array.

Reply:
Revised.